client/src/components/atoms/style/AlertDialog.scss

```diff
@@ -0,0 +1,37 @@
+.alert-display-text {
```

Comment:
Can we either append the class names with `alert-box`? I think it'll make the styling more consistent with other components. Then we can write the scss like this: 

```
.alert-box {
  &-display-text {
  }

  &-leave-button {
  }

  etc.
}
```

Reply:
Or we could write the scss like this:
```
.alert-box {
  .alert-display-text { }
  .alert-leave-button {}
}
```

So that we're only selecting `alert-display-text` elements who have ancestors with `alert-box`

Reply:
Updated.

Reply:
hm it looks fine when my browser is at 1440px and 100% zoom.

Reply:
yikes ok not sure what's going on

Reply:
LOL can we fix the width at 513px and call it a day?